Dockerfile

```diff
@@ -186,7 +186,7 @@ RUN cp /tensorflow/bazel-bin/native_client/generate_trie /DeepSpeech/native_clie
 
 # Install TensorFlow
 WORKDIR /DeepSpeech/
-RUN pip install tensorflow-gpu==1.12.0rc2
+RUN pip install tensorflow-gpu==1.12.0
```

Comment:
But that should likely be fixed?

Reply:
No, it'll be broken regardless. 1.13 requires new versions of CUDA/cuDNN/NCCL/etc, so installing 1.13 or not doesn't make any difference.

Reply:
Well, we can update that `FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04`

Reply:
Probably. Then we also have to update the apt-get package names/versions, etc. I haven't tested.